lib/crowbar/client/command/upgrade/nodes.rb

```diff
@@ -34,8 +34,16 @@ def execute
             request.process do |request|
               case request.code
               when 200
-                say "Successfully triggered the upgrade of the nodes. " \
-                  "Query the upgrade status to follow the process with 'crowbarctl upgrade status'."
+                case args.component
+                when "all"
+                  say "Successfully triggered the upgrade of the nodes. "
+                when "controllers"
+                  say "Successfully triggered the upgrade of the controller nodes. "
+                else
+                  say "Successfully triggered the upgrade of compute node #{args.component}. "
```

Comment:
Do not mention *compute* here. It's true we're only thinking about compute nodes, but better to not limit us for possible changes